lightning/src/events/mod.rs

```diff
@@ -774,10 +774,10 @@ pub enum Event {
 		/// Information for claiming this received payment, based on whether the purpose of the
 		/// payment is to pay an invoice or to send a spontaneous payment.
 		purpose: PaymentPurpose,
-		/// The `channel_id` indicating over which channel we received the payment.
-		via_channel_id: Option<ChannelId>,
-		/// The `user_channel_id` indicating over which channel we received the payment.
-		via_user_channel_id: Option<u128>,
+		/// The `(channel_id, user_channel_id)` pairs over which the payment was received.
+		///
+		/// This will be an incomplete vector for MPP payment events created/serialized using LDK version 0.1.0 and prior.
+		via_channel_id_pairs: Vec<(ChannelId, Option<u128>)>,
```

Comment:
Also standardize on `inbound_*` perhaps. Via is more broad. Although pre-existing also.

Reply:
Thanks for the suggestion!
Updated the name to `inbound_channel_ids` in [pr3655.12](https://github.com/shaavan/rust-lightning/commits/pr3655.12)